### DISCUSSION TITLE
不同语言下各个分类项categories或各个系列项series的中英文名称如何定义？

### DISCUSSION THREAD

**Discussion by KinShin** [HEIGE-PCloud/DoIt#1395] [Q&A] [answered]
我刚刚开始使用 hugo，发现 DoIT 是最符合我需求的主题，非常感谢作者们的贡献。

但是我阅读了主题的在线文档，还是没法解答我的疑惑，所以来这里提问。

我了解到“分类”或者“系列”本身在不同语言下可以通过 [[languages.en.menu.main]] 和 [[languages.zh-cn.menu.main]] 来指定显示名称。

但是“分类”下的各个“分类项”在不同语言下的名称该如何指定呢？

比如这篇文章 https://hugodoit.pages.dev/zh-cn/theme-documentation-basics/ 的分类在 index.zh-cn.md 和 index.en.md 下都是 categories: ["documentation"]，但是在中文下显示的是“收录于 类别 **_文档_**”，在英文下显示的是“included in category **_Documentation_**”。

感谢不吝赐教。

**Comment by CXwudi** [comment, marked as answer]
这是我DoIt最喜欢的点之一，所有的categories，series，还有tags，都是通过专门的文件夹建立来确保i18n工作，比如说`documentation`类别就是在[`content/categories`](https://github.com/HEIGE-PCloud/DoIt/tree/2349f8f660d5a367d44498f6696515a1d335a0be/exampleSite/content/categories)中定义的，这样在显示文章的时候就可以调用相应的i18n string

顺便说一下作者profile也有i18n，在官方例子里没有但我自己的博客里有个[例子](https://github.com/CXwudi/personal-blog-source/blob/main/data/authors/CXwudi.toml)

**Reply by KinShin** [reply]
多谢指点。

原来是这样实现的。对Hugo，或者说DoIt有了更深的理解。

原来主题里包含了演示，看来光看文档还不够。
